go.mod

```diff
@@ -24,22 +24,22 @@ require (
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.32.0
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.32.0
 	go.opentelemetry.io/otel/sdk v1.32.0
-	golang.org/x/sys v0.27.0
-	google.golang.org/grpc v1.68.0
+	golang.org/x/sys v0.28.0
+	google.golang.org/grpc v1.68.1
 	google.golang.org/protobuf v1.35.2
-	k8s.io/api v0.31.3
-	k8s.io/apimachinery v0.31.3
-	k8s.io/client-go v0.31.3
-	k8s.io/component-base v0.31.2
+	k8s.io/api v0.31.4
+	k8s.io/apimachinery v0.31.4
+	k8s.io/client-go v1.5.2
```

Comment:
Looks like the latest version is v0.31.4 https://pkg.go.dev/k8s.io/client-go/kubernetes

Reply:
terrific catch @ElijahQuinones <3

Yeah this v0 version Go-community tradition for "We do not make compatibility promises" is highly annoying.